docs/RELEASE_POLICY.md

```diff
@@ -92,7 +92,7 @@ v3.1 will be patched since were the last two Firecracker releases and less than
 | ------: | -----------: | -----------: | ------------------: | :------------------------------ |
 |   v1.10 |   2024-11-07 |      v1.10.1 |          2025-05-07 | Supported                       |
 |    v1.9 |   2024-09-02 |       v1.9.1 |          2025-03-02 | Supported                       |
-|    v1.8 |   2024-07-10 |       v1.8.0 |          2025-01-10 | Supported                       |
+|    v1.8 |   2024-07-10 |       v1.8.0 |          2025-01-10 | 2025-01-10 (end of 6mo support) |
```

Comment:
LGTM, but commit message and PR should read unsupported, not deprecated. `s/deprecated/unsupported/g`

Reply:
fixed

Reply:
Thanks!